docs/hugo/content/introduction/multitenant-deployment.md

```diff
@@ -0,0 +1,78 @@
+# Deploying ASO v2 in multi-tenant mode
+
+The current release version of the ASO deployment YAML installs the operator in single-tenant mode:
+it's deployed in the `azureserviceoperator-system` namespace with one set of Azure credentials, and manages resources in any namespace in the cluster.
```

Comment:
Perhaps tweak the wording/formatting to emphasize what's different about single tenant mode?
```suggestion
The current release version of the ASO deployment YAML installs the operator in _single-tenant_ mode:
deployed in the `azureserviceoperator-system` namespace with a _single set of Azure credentials_, and manages resources in _any namespace_ in the cluster.
```